new-frontiers-14/frontier-station-14
# Deep fryer recipes, fried chicken, nuggets, doughnuts, Nyano drinks

## About the PR



Adds deep fryer recipes and entries for them in the food guidebook.

Adds recipes for battered chicken and battered chicken cutlets, which can be deep fried into fried chicken and chicken nuggets, respectively.

Edits the "Pizza" construction graph to include a step to add butter and roll it out to simulate laminating dough for pastry.  This pastry dough can then be cut into raw pastry slices, which can be rolled into doughnuts, optionally combined with berry juice and sugar in a Food-O-Mat, deep fried, and optionally topped with a variety of toppings.

Restores recipes for Nyanotrasen drinks (taken off the head of master, all recipes located in their respective Nyanotrasen folders, and many thanks to the original contributors).

## Why / Balance


More involved cooking, better use of the deep fryer.

Nyanotrasen drinks: we have the sprites, why not?

## How to test


1. Make some doughnuts from scratch.  Start with dough, roll it out, add a stick of butter, then roll out the buttered dough.  Slice the resulting pastry dough into slices, and roll each out into a doughnut, then pop it into the fryer until cooked.
2. Burn a doughnut.  Start with an uncooked doughnut, keep it in the fryer.  It should cook into a plain doughnut before burning eventually.
3. Glaze a few cooked doughnuts in the Food-O-Mat.  Refer to the guidebook for recipes (generally fruit juices, sugar if using powders [tea powder, cocoa powder]).
4. Make each of the new drinks.

## Media


![image](https://github.com/user-attachments/assets/5cc63859-f92a-4ed0-a0ea-a7b0f97bab30)

## Requirements



## Breaking changes


**Changelog**


:cl:
- add: Fried chicken and chicken nuggets can now be made in the deep fryer.
- add: Donuts can now be made from buttering and rolling out flat dough, deep fry them and glaze them in the Food-O-Mat.
